Makefile

```diff
@@ -19,6 +19,7 @@ sf:
 # Run all of the project-level tests, followed by app-level tests
 test: assets
 	$(BIN)/mocha $(shell find api -name '*.test.coffee')
+	$(BIN)/mocha $(shell find api -name '*.test.js')
 	$(BIN)/mocha $(shell find client -name '*.test.coffee')
```

Comment:
will 22 conflict with 24?  we have jest running on all the .js tests at the moment

Reply:
oh nvmd i see it is the api side